lib/mutant.rb

```diff
@@ -200,7 +200,14 @@ class Config
       fail_fast:         false,
       includes:          EMPTY_ARRAY,
       integration:       Integration::Null,
-      isolation:         Mutant::Isolation::Fork,
+      isolation:         Mutant::Isolation::Fork.new(
+        devnull: ->(&block) { File.open(File::NULL, File::WRONLY, &block) },
```

Comment:
Ahh good point. I don't think I'd ever seen that error. I do think it should be possible to change the access mode if you are restricting it further but I can see why they chose to do that.